__tests__/app.test.js

```diff
@@ -44,6 +41,35 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: returns and array with the selected article.", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body.article)).toHaveLength(8);
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+
+  test("400: if article_id is not a Number, returns invalid format", () => {
```

Comment:
Good error test! What about if I tried to request article number 1000? What should happen?